src/tests/server.ts

```diff
@@ -3,6 +3,10 @@ import { defaultRouter } from "./default.router";
 import { compiledRouter } from "./compiled.router";
 import { redisRouter } from "./redis.router";
 
+export { cache as defaultCache } from "./default.router";
+export { cache as compiledCache } from "./compiled.router";
+export { cache as redisCache } from "./redis.router";
+
```

Comment:
Why do we need to re-export the middlewares here?

Reply:
Limit exposure of router classes. Reduce coupling. Seemed cleaner but not a big deal.

Reply:
Fair enough